docs/agents/sematext-agent/packages/data.md

```diff
@@ -0,0 +1,13 @@
+title: Package Data
+description: Package Data collected by Sematext Agent
+
+Package payloads are comprised of various fields as summarized in the table. Sematext Agent currently supports Python, Node.js and Debian-based packages.
+
+| Name          | Description |
+| --------------|-------------|
+| name          | Represents the package name |
+| version       | Is the version linked to the particular package |   
+| type          | Designates the type of the package (e.g. Python) |
+| architecture  | Determines the architecture the package was built (32/64 bits) |
```

Comment:
=> Determines the architecture for which the package was built

Reply:
Tried to find the link you posted, but the convo got lost and no luck searching in Github :(